erdpy_wallet/keyfile.py

```diff
@@ -69,10 +69,10 @@ def load_from_key_file_object(keystore: Dict[str, Any], password: str) -> Tuple[
     return address_bech32, secret_key
 
 
-def convert_to_keyfile_object(secret_key: bytes, pubkey: bytes, password: str, randomness: Union[None, IUserWalletRandomness]) -> Dict[str, Any]:
-    salt = os.urandom(32) if randomness is None else randomness.get_salt()
-    iv = os.urandom(16) if randomness is None else randomness.get_iv()
-    uid = str(uuid4()) if randomness is None else randomness.get_id()
+def convert_to_keyfile_object(secret_key: bytes, pubkey: bytes, password: str, randomness: Union[None, IUserWalletRandomness], address_prefix: str) -> Dict[str, Any]:
```

Comment:
does `address_prefix` refer to the HRP part of the bech32 address? If so, you could rename it

Reply:
Renamed to `address_hrp`.